src/main/resources/swagger/api-docs.yaml

```diff
@@ -533,6 +547,32 @@ paths:
       summary: Query to see if you're a library curator
       operationId: isCurator
 
+  /api/library/{workspaceNamespace}/{workspaceName}/discoverableBy:
+    put:
```

Comment:
Also ... why isn't there a `POST` for this? Initially, one would be `POST`ing a new group and subsequent calls would be making a `PUT` request.